headers/augment_headers.py

```diff
@@ -289,6 +293,13 @@ def add_header_content(
         action="store_true",
         help="add alias declarations",
     )
+    parser.add_argument(
+        "-ad",
```

Comment:
I didn't know you could have two-character short flags? Won't this interfere with the usual meaning of "-ad" which is "-a -d"? Let's just remove this and only leave the long name, argparse can handle that.